lib/CMakeLists.txt

```diff
@@ -5,6 +5,6 @@ add_library(acme_lw ${source})
 install (TARGETS acme_lw DESTINATION lib)
 install (FILES acme-exception.h acme-lw.h DESTINATION include)
 
-target_include_directories(acme_lw PUBLIC ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/../internal )
+target_include_directories(acme_lw PUBLIC ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/../internal ${OPENSSL_INCLUDE_DIR})
 
 target_link_libraries(acme_lw PRIVATE crypto)
```

Comment:
I tried building this on a Mac and it wouldn't link until I replaced 'crypto' with ${OPENSSL_LIBRARIES}. You didn't need to do that, or equivalent?

Reply:
Interesting. No, I didn't have to. My CMake install found OpenSSL via homebrew:

`OPENSSL_CRYPTO_LIBRARY           /usr/local/opt/openssl/lib/libcrypto.dylib
 OPENSSL_INCLUDE_DIR              /usr/local/opt/openssl/include
 OPENSSL_SSL_LIBRARY              /usr/local/opt/openssl/lib/libssl.dylib`

So I suspect it worked by accident. I'll make the change to use the FindOpenSSL macro as you suggest.

Reply:
Thanks, that would be great. I used ports rather than brew and I assume that's the difference.